docs/contributing/code-style/csharp.md

```diff
@@ -163,13 +163,49 @@ services.TryAddEnumerable(ServiceDescriptor.KeyedTransient<IMyService, FirstMySe
 In multi-inject scenarios you are especially in danger of adding the same implementation multiple
 times and having them all be used.
 
-### Dependency Groups
+### Dependency groups
 
 Consider creating [dependency groups][dependency-groups]. This gives other teams a nice interface to
 register your group when they depend on it. If you've used
 [`TryAdd`](#use-tryadd-overloads-on-iservicecollection) overloads it won't matter how many times
 your group of services is added to the collection.
 
+## GUID generation
+
+Always use `CoreHelpers.GenerateComb()` for entity IDs instead of `Guid.NewGuid()`. COMB GUIDs
+prevent SQL Server index fragmentation by incorporating timestamp information for sequential
+ordering.
```

Comment:
🎨 Can you provide a link or two on what these even are? I regularly find folks not aware that this is a thing.